packages/pipeline/src/entities/ohlcv_external.ts

```diff
@@ -0,0 +1,20 @@
+import { Column, Entity, PrimaryColumn } from 'typeorm';
+
+@Entity({ name: 'ohlcv_external', schema: 'raw' })
+export class OHLCVExternal {
+    @PrimaryColumn() public exchange!: string;
+    @PrimaryColumn() public from_symbol!: string;
+    @PrimaryColumn() public to_symbol!: string;
+    @PrimaryColumn() public start_time!: number;
+    @PrimaryColumn() public end_time!: number;
+    
+    @Column() public open!: number;
+    @Column() public close!: number;
+    @Column() public low!: number;
+    @Column() public high!: number;
+    @Column() public volume_from!: number;
```

Comment:
We usually use `camelCase` for properties in TypeScript. Can we change this so it looks like the other entities? You will need to specify the `name` in the `@Colum` decorator since it is different than the property name.

For example:

```ts
@Column({name: 'volume_from'})
public volumeFrom!: number;
```

Reply:
yeah